packages/api/src/controllers/farmExternalIntegrationController.js

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright 2019, 2020, 2021, 2022 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+const farmExternalIntegrationController = {
```

Comment:
@seinnlee I think you worked on this. I'm not sure any of these endpoints are necessary, since it seems like we handle these use cases at at the database level. Let me know what you think.

Reply:
Yeah we don't need them. You can delete them if you already have the same thing somewhere else.